src/app/connections/services/connections.service.ts

```diff
@@ -24,19 +24,21 @@ export class ConnectionsService {
       .append("programmeName", snapshot.tableFilters?.programmeName || "");
 
     if (snapshot.filter === "discrepancies") {
+      let noneSelected = !(snapshot.tableFilters?.dbcs || snapshot.tableFilters?.tisDesignatedBodies)
```

Comment:
I realise I could drop the ! on both of these for simpler logic, but I went for what I thought was the clearer naming & intent option

Let me know if "atLeastOneSelected" is preferable